### PR TITLE
Make Tutorial how to mint from Mobile wallet on KodaDot #915

### LINKED ISSUE #915: Make Tutorial how to mint from Mobile wallet on KodaDot
We would like to show people how to mint on KodaDot straight from mobile
Best if you are English native speaker with good accent and amazing studio mic, tutorial should be in ENGLISH
Tutorial should have this steps
- [ ] brief introduction -- onboarding voice video
- [ ] setup wallet for first timers
- [ ] how to mint your stuff from mobile, use simple mint probably
- [ ] collect stuff
- [ ] show screens like spotlight and series

### PR DESCRIPTION
**Pull request by AshutoshSingh00001**
Thank you for your contribution to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz/).

👇 __ Let's make a quick check before the contribution.

## PR Type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #915
## Did your issue had any of the "$" label on it?
- [x] My DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=158VTDxZJidzsMcAdh7h828domveRvpmk1SCxwLUEHhcgY4v&usdamount=0&donation=true)

### CODE REVIEW

**Comment by roiLeo** [comment]
label blocked by #7208

**Comment by vikiival** [comment]
cc @helloitsdamsky

**Comment by AshutoshSingh72** [comment]
> cc @helloitsdamsky

@yangwao cc

**Comment by helloitsdamsky** [comment]
hey @AshutoshSingh72, thanks for your contribution.

Unfortunately, the video you provided is:
- hard to understand
- somewhat sped up?
- hard to follow
- too long

Ideally, what we are aiming regarding quality is to have something like this:

- https://www.youtube.com/watch?v=SKdx4pTgL04&t=16s

Key points:

- Slow and clear pronunciation
- Easy to understand
- Easy to follow
- Ideally under 5 min
- Work on post-production 
- Explain your steps in understandable way for a non-tech user

Therefore, we’ll be looking forward for your next contributions, following the guidelines

**Review by helloitsdamsky — CHANGES_REQUESTED** [review]
As per https://github.com/kodadot/nft-gallery/pull/7303#issuecomment-1732634303 this can be closed

**Comment by AshutoshSingh72** [comment]
> As per [#7303 (comment)](https://github.com/kodadot/nft-gallery/pull/7303#issuecomment-1732634303) this can be closed

I'll make changes today i'll follow your guidelines and make it perfect accordingly

**Comment by AshutoshSingh00001** [comment]
> hey @AshutoshSingh72, thanks for your contribution.
> 
> Unfortunately, the video you provided is:
> 
> * hard to understand
> * somewhat sped up?
> * hard to follow
> * too long
> 
> Ideally, what we are aiming regarding quality is to have something like this:
> 
> * https://www.youtube.com/watch?v=SKdx4pTgL04&t=16s
> 
> Key points:
> 
> * Slow and clear pronunciation
> * Easy to understand
> * Easy to follow
> * Ideally under 5 min
> * Work on post-production
> * Explain your steps in understandable way for a non-tech user
> 
> Therefore, we’ll be looking forward for your next contributions, following the guidelines

Check @helloitsdamsky updated

**Comment by helloitsdamsky** [comment]
> Check @helloitsdamsky updated

For official tutorials that are onboarding our users to platform, we need native speakers or people who speak for very clearly. Unfortunately, i still can’t understand 20% of what you are saying.

I recommend making a script and using https://elevenlabs.io/ for clear and easy to understand output.

**Comment by AshutoshSingh72** [comment]
> > Check @helloitsdamsky updated
> 
> For official tutorials that are onboarding our users to platform, we need native speakers or people who speak very clearly. Unfortunately, i still can’t understand 20% of what you are saying.
> 
> I recommend making a script and using https://elevenlabs.io/ for clear and easy to understand output.

I'll try to enhance my speech by using that

**Comment by AshutoshSingh00001** [comment]
> > Check @helloitsdamsky updated
> 
I'll try to enhance my speech by using that

**Comment by AshutoshSingh00001** [comment]
> > > Check @helloitsdamsky updated
> 
> I'll try to enhance my speech by using that

Cc @helloitsdamsky i made it using eleven labs ai under 60 seconds

**Comment by helloitsdamsky** [comment]
Let’s please close this, this is far from quality we’d like to present kodadot with. We will need to create guidelines for content creators, otherwise both parties are loosing time.
